### PR TITLE
DOCUMENTATION: frequencies, meshing, copyright

### LINKED ISSUE #168: Documentation improvements: meshing, frequencies
We've discovered/remembered a couple important things that are worth noting in the documentation. We'll put together a list here and then incorporate into the documentation.

- When choosing the number of frequencies, you should consider
  - If the response will be nonlinear (e.g., constraints exist, nonlinear kinematics), you will need higher frequency components 
  - Must always have 2x the incoming wave frequency, as the power absorption will take place at this frequency

### LINKED ISSUE #184: wec.solve() returns only one type of output, not 3 as described
function comment says it returns 3 output        
        res_fd
            Dynamic responses in the frequency-domain.
        res_td
            Dynamic responses in the time-domain.
        res
            Results produced by :py:func:`scipy.optimize.minimize`.

### LINKED ISSUE #195: docs: "copyright" appears twice
image

### PR DESCRIPTION
**Pull request by ryancoe**
Closes #168
Closes #195 
Closes #184

### CODE REVIEW

File: docs/source/theory.rst [hunk 1]
```diff
@@ -75,6 +75,16 @@ At first the solution will not be correct, but as the optimization algorithm ite
...
 ------------------
 
+Frequencies
+^^^^^^^^^^^
+The solution to :eq:`optim_prob` will be found by projecting the time-domain trajectories into the frequency domain (see further discussion in :doc:`implementation`).
+To control this process, the user must specify an equally spaced frequency array (see :py:func:`wecopttool.frequency`).
+When creating this frequency array, consider
+
+    * Exciting frequencies of the wave and, if you expect a nonlinear response, superharmonics of the exciting frequencies
+    * Power absorption occurs at two times the exciting frequencies
```
Comment (cmichelenstrofer) [inline docs/source/theory.rst:85]: This might be as important as we thought. Up to you whether to leave or remove or modify.

**Review by cmichelenstrofer — APPROVED** [review]

**Comment by cmichelenstrofer** [comment]
@ryancoe should we merge this?

File: docs/source/theory.rst (hunk 1 above)
Comment (ryancoe) [reply docs/source/theory.rst:85]: probably should be removed

**Comment by ryancoe** [comment]
I think check whether we're fully addressing #168. Ok if not, but just note this properly. Then Ok to merge. My view is that meshing and irregular frequencies are Capytaine issues. We should refer users to Capytaine instead of trying to solve that ourselves.

**Comment by ryancoe** [comment]
@michaelcdevin - Can you give me a hand with getting this documentation to build? I'm having trouble with link checks I think. Here's the error that I get locally (different from the tests above):

```
sphinx.errors.SphinxWarning: /Users/rcoe/Documents/WecOptTool/wecopttool/docs/source/api_docs/wecopttool.WEC.solve.rst:1:broken link: https://github.com/HIPS/autograd (HTTPSConnectionPool(host='github.com', port=443): Max retries exceeded with url: /HIPS/autograd (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)'))))
```

**Comment by michaelcdevin** [comment]
This isn't just you @ryancoe, it looks like this is happening when I test the main branch locally too. This doesn't occur in the CI, so it's probably a proxy issue with GitHub links. I'll look into this more when I have a chance. In the meantime, you can workaround this by adding `https://github.com/*` to the [`linkcheck_ignore` section in `conf.py`](https://github.com/sandialabs/WecOptTool/blob/9380543abd3f2ed5e7b07afef651bcb070a5ca4f/docs/source/conf.py#L60).

**Pull request merged**